### DISCUSSION TITLE
Emoji symbols aren't appearing in Debian 11

### DISCUSSION THREAD

**Discussion by gianfelicevincenzo** [starship/starship#2964] [Q&A]
Hi. I have already debian 11 and i have installed nerd-fonts...but unsuccessfully, because (again) starship doesn't use the right types of fonts (even though I installed them and they work!)

How can I solve?

(in this case, script perl)...

![2021-08-06_15-52](https://user-images.githubusercontent.com/30395310/128520876-2d1ae657-e94c-4354-8997-ea949d335c9d.png)

**Comment by vladimyr** [comment]
Perl module doesn't use Nerd font but emoji instead https://starship.rs/config/#perl

Please install Noto emoji font as described here: https://starship.rs/faq/#why-don-t-i-see-a-glyph-symbol-in-my-prompt

---

I would kindly ask you to avoid shouting in discussion titles as there is absolutely no need for such behavior.
